Exiled.Events/EventArgs/Scp049/DoctorSenseEventArgs.cs

```diff
@@ -0,0 +1,78 @@
+// -----------------------------------------------------------------------
+// <copyright file="DoctorSenseEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp049
+{
+    using API.Features;
+    using Interfaces;
+    using Mirror;
+    using PlayerRoles.PlayableScps.Scp049;
+
+    /// <summary>
+    ///     Contains all information before SCP-049 sense is sent to client.
+    /// </summary>
+    public class DoctorSenseEventArgs : IPlayerEvent, IDeniableEvent
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DoctorSenseEventArgs"/> class with information before SCP-049 sense is sent to client.
+        /// </summary>
+        /// <param name="scp049"> Scp049 <see cref="Player"/>. </param>
+        /// <param name="target"> Target <see cref="Player"/>. </param>
+        /// <param name="senseAbility"> Doctor's <see cref="Scp049SenseAbility"/> ability. </param>
+        /// <param name="isAllowed"><inheritdoc cref="DoctorSenseEventArgs.IsAllowed"/></param>
+        public DoctorSenseEventArgs(Player scp049, Player target, Scp049SenseAbility senseAbility, bool isAllowed = true)
+        {
+            Player = scp049;
+            Target = target;
+            SenseAbility = senseAbility;
+            IsAllowed = isAllowed;
+            Cooldown = 5f;
+            Duration = 20f;
```

Comment:
Could we not hsrdcoded them

Reply:
Yeah that's fair when I'm home ill do that